### PR TITLE
#6 - Added basic project info to README.md

### PR DESCRIPTION
**Pull request by olegmoz**

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#8](https://github.com/artipie/nuget-adapter/issues/8) is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/7ba6a916-bcd6-45a4-89a3-e1c1df0a203b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+  <groupId>com.artipie</groupId>
+  <artifactId>nuget-adapter</artifactId>
+  <version>[...]</version>
+</dependency>
+```
+
+Save package file in `.nupkg` format to storage. 
+Then, you make an instance of `Repository` class with your storage
+as an argument.
+Finally, you instruct `Repository` to add the package to repository:
```
Comment (olenagerasimova) [inline README.md]: @olegmoz I'd suggest to remove "you" from both sentences: looks like you have imperative mood here, so no need for pronouns.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+
+```xml
+<dependency>
+  <groupId>com.artipie</groupId>
+  <artifactId>nuget-adapter</artifactId>
+  <version>[...]</version>
+</dependency>
+```
+
+Save package file in `.nupkg` format to storage.
```
Comment (olenagerasimova) [inline README.md]: @olegmoz being new with NuGet this sentence looks unclear: 

> package file in `.nupkg` format 

you mean this special NuGet ZIP file renamed with the `.nupkg` extension? And what storage?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,52 @@
-# nuget-adapter
-Nuget Artipie adapter
+<img src="https://www.artipie.com/logo.svg" width="64px" height="64px"/>
```
Comment (olenagerasimova) [inline README.md]: @olegmoz also I'd suggest to add icons for EO, Rultor, Idea etc

File: README.md (hunk 1 above)
Comment (olegmoz) [reply README.md]: Fixed that, thank you!

File: README.md (hunk 2 above)
Comment (olegmoz) [reply README.md]: Added some details, hope it is more clear now.

File: README.md (hunk 3 above)
Comment (olegmoz) [reply README.md]: I think other icons such as Rultor, build status etc should be added as they are added to the project later. And for other stuff - I would rather avoid information noise on front page of the project.

**Comment by olenagerasimova** [comment]
@g4s8 this PR look good to me, please, take a look/merge

**Comment by yegor256** [comment]
@olegmoz done, thanks

**Comment by 0crat** [comment]
Job was finished in 26 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**